files/dhcp/graphserviceurl

```diff
@@ -0,0 +1,11 @@
+case $reason in
+    BOUND|RENEW|REBIND|REBOOT)
+        if [ -f "/etc/sonic/graph_service_url" ] && [ -n `cat /etc/sonic/graph_service_url` ]; then
+            exit 0
+        elif [ -n "$new_minigraph_service_url" ]; then
+            echo $new_minigraph_service_url > /etc/sonic/graph_service_url
+        else
+            echo "disabled" > /etc/sonic/graph_service_url
```

Comment:
>graph_service_url [](start = 41, length = 17)

rename to updategraph.conf since it is the configuration file for the upgradegraph srevice.

also the conf file format needs to defined. usually something like below

# whether to enable/disable update graph service
enable=true|false

# url
url = xxx xxx xxx

it is possible that the dhcp provide you multiple url to download.